astropy/units/tests/test_format.py

```diff
@@ -379,6 +379,12 @@ def test_flexible_float():
     assert u.min._represents.to_string('latex') == r'$\mathrm{60\,s}$'
 
 
+def test_fits_to_string_function_error():
+    """Test if will raise a TypeError if incorrect unit type."""
```

Comment:
This maybe is a good place to note why we are testing this, perhaps
"""Test function raises TypeError on bad input.

This instead of returning `None`, see gh-11825.
"""